web_app/derisk_alert/tasks.py

```diff
@@ -70,3 +72,21 @@ def ekubo_order_book():
                 f"With token pair: {token_a} and {token_b} something happened: {exc}"
             )
             continue
+
+
+@app.task(name="haiko_order_book")
```

Comment:
We should move the whole Haiko part into `data_handler`.

Reply:
@lukaspetrasek I will do that later with all order_books. It would be better to do it by batch because there are lot of dependencies in imports and tasks.